### ISSUE TITLE
Animation Call #9

### ISSUE THREAD

**Issue by Zaxounette** [btcpayserver/organization#24]
```meta
Date: 2021-02-24
Time: 5:00pm
Timezone: UTC
Duration: 1h
UTCTime: 2021-02-24 17:00 UTC
```

### Agenda

1. Final character renders
2. Final speech text iterations for characters and narrator before lock-in.

For reference: [Previous call (N°8)](https://github.com/btcpayserver/organization/issues/20)

### Check your timezone

https://everytimezone.com/s/fcc4fa10

### Join the call

https://meet.jit.si/BTCPayAnimationCall

### Calendar invite

Subscribe to the BTCPay Server calendar. More info [here](https://github.com/btcpayserver/organization#calendar).

**Comment by Zaxounette** [comment]
Animation Call N°9 notes.

Attendees: @britttttk , @Cosmicsound , @webworthy , @Zaxounette 

Webworthy shows us shot iterations and modifications since last week on the [StoryBoard](https://app.milanote.com/1LbQzL14owMH7B/storyboard?p=nZfM3RpnrSC).

Most notable are the changes in the street scene, SCENE 4 SHOT 3 and beyond. The changes are intended to emphasis on the Old Architecture being mixed with the New BTCPay Server to produce the future.

An update on the music is also presented.
The piano intro could be shortened to accommodate for the short animation intro that is expected.
Full timing TBD once animation is advanced sufficiently.

Most of the call is concentrated on the lock-in of most of the speaking (narration and sync-speach) strings.
Are locked-in strings up to SCENE 3 SHOT 1 **included**. 
See [Storyboard](https://app.milanote.com/1LbQzL14owMH7B/storyboard?p=nZfM3RpnrSC) for more information.

The last strings will be homework for Cosmic, Britt and Zax and locked-in next call.

HOMEWORK: 
@Cosmicsound , @britttttk , @Zaxounette : Iterate on the last strings in layman terms. Last iterations before lock-in.
@webworthy : Continu iteration on scenes and shots.

Next call: Wednesday 3rd March, 17:00 UTC.

**Comment by uhlhosting** [comment]
Here #CosmicSound so that someone else is not confused 🍡
